lib/sandbox.js

```diff
@@ -161,8 +162,7 @@ function init ( ) {
   }
 
   function unitsLabel ( ) {
-    if (sbx.units == 'mmol') return 'mmol/L';
-    return 'mg/dl';
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/761330323/issues/source?bid=2452379&fileBranchId=2452987#l165)